src/composables/__tests__/loader.test.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @vitest-environment happy-dom
+ */
+
+import {
+  cleanup,
+  fireEvent,
+  render,
+  screen,
+  waitForElementToBeRemoved
+} from '@testing-library/vue'
+import { afterEach, test } from 'vitest'
+
+import TestItemWithLoader from './test-item-with-loader.vue'
```

Comment:
Easier to read if the component is inlined where it's tested? It shouldn't be used anywhere else. Just a suggestion.

Reply:
Let's try that out in a subsequent PR? I tried first to do that but couldn't make it work properly.